spec/features/content_item_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "Content Item Details", type: :feature do
```

Comment:
Integration tests exercise all the stack without being exhaustive in the tests; so my feeling is that it would be ok to check for the title of the content item to be ok. 
I like the `Then I am on the content item detail page.`. But I would not rely on the browser URL as that would also pass with a blank page. But, feel free to follow the approach you think is better.